plugins/pagerduty/komand_pagerduty/actions/delete_user_by_id/action.py

```diff
@@ -18,23 +18,15 @@ def __init__(self):
     def run(self, params={}):
         """Delete user"""
 
-        if not params["id"]:
-            raise Exception("id not provided")
+        email = params.get("email")
+        user_id = params.get("id")
 
-        user = pypd.User.find_one(id=params["id"])
-        if not user:
-            return {
-                "success": False,
-                "user": {},
-            }
+        if email is None or user_id is None:
+            self.logger.warning("Please ensure a valid 'email' and 'id' is provided")
+            raise PluginException(
+                cause="Missing required paramaters", assistance="Please ensure a valid 'email' and 'id' is provided"
+            )
```

Comment:
if they are both defined as required in the plugin spec file then this is redundant

Reply:
I removed all of  these redundant error raises